app/models/manageiq/providers/vmware/infra_manager/vm_or_template_shared/scanning.rb

```diff
@@ -29,10 +43,6 @@ def perform_metadata_sync(ost)
     sync_stashed_metadata(ost)
   end
 
-  def validate_smartstate_analysis
-    validate_supported_check("Smartstate Analysis")
```

Comment:
The intent of this method is to eliminate code duplication of the archived? and orphaned? checks. Why are you adding duplicate code in so many places?